src/test/lsipii/User/StatusInfo/Write.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+from enum import Enum
+
+from converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class StatusInfoValue(str, Enum):
+    SENT = ("SENT",)
+    PROCESSING = ("PROCESSING",)
+    WAITING_FOR_COMPLETION = ("WAITING_FOR_COMPLETION",)
+    READY = ("READY",)
```

Comment:
Same suggestion as in the other definition:
```suggestion
    SENT = "SENT"
    PROCESSING = "PROCESSING"
    WAITING_FOR_COMPLETION = "WAITING_FOR_COMPLETION"
    READY = "READY"
```